data/text/items.ts

```diff
@@ -846,7 +846,10 @@ export const ItemsText: {[id: IDEntry]: ItemText} = {
 	},
 	icyrock: {
 		name: "Icy Rock",
-		shortDesc: "Holder's use of Hail lasts 8 turns instead of 5.",
+		shortDesc: "Holder's use of Snow lasts 8 turns instead of 5.",
```

Comment:
Should be "Snowscape" to be consistent with the other weather rock descriptions.

Reply:
True, didn't even think about it that way.